### ISSUE TITLE
[v10] Submit button with text

### ISSUE THREAD

**Issue by musshush** [auth0/lock#477]
We need the 'submit' button with text instead of just the icon, but v10 does not seem to have any option to set text for the button. And this text needs to be translated as well.

Is there a way that I'm missing?
Or, can we have this functionality?

**Comment by gnandretta** [comment]
Hi, currently is not possible to show some text in the submit button. We'll add this feature in the future.

**Comment by TranquilMarmot** [comment]
If you _really_ want this right now with v10 there is some hacky CSS you do around it...

```css
button.auth0-lock-submit:before {
    content: "Sign In";
    font-size: 20px;
}

button.auth0-lock-submit span {
    display: none !important;
}
```

**Comment by gnandretta** [comment]
Implemented in #524 and released in `v10.2.0`. The text for the submit button can be configured based on the screen with the following language keys:`loginSubmitLabel`, `signUpSubmitLabel` and `forgotPasswordSubmitLabel`.
